examples/odesolver.jl

```diff
@@ -0,0 +1,46 @@
+#=
```

Comment:
I don't see in which notebook this is included?

Reply:
Forgot to put this somewhere. I can recover this for the slides?